### PR TITLE
Added step CSS Handle to LoginContent Component

### PR DESCRIPTION
**Pull request by JNussens**
#### What is the purpose of this pull request?

Added step CSS Handle to LoginContent Component

#### What problem is this solving?

Style each step accordingly

#### How should this be manually tested?
With the pull-request updates, inspect the login content div and see the step handle changing accordingly.

#### Screenshots or example usage
http://thirdbranch--storeido.myvtex.com/
image

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by rafarubim** [comment]
Hi @JNussens! Thanks for the contribution to our app. I understand the changes you're proposing, but I'm a little worried about showing the `step` in a CSS handle. The current `STEPS` implementation is kind of cumbersome and that would make it even harder for us to refactor it later (without making a breaking change).

Can you please elaborate on the problem you want to solve? Maybe we can find another way around it.

**Comment by JNussens** [comment]
@rafarubim We could also explain why this property exists and warn the user about this possible issue. What do you think? We really need this.

**Comment by rafarubim** [comment]
@JNussens even with a warning or deprecation notice, that will make breaking changes in case of a refactor. What about naming each step as a different CSS handle? It could be mapped somehow like this:
```js
const styleByStep = [styles.emailVerificationContainer, styles.emailAndPasswordContainer, ...]

<div className={styleByStep[step]} />
```
Then at least the steps are not exposed directly to the interface

**Comment by klzns** [comment]
@JNussens can you provide a linked workspace?

**Comment by klzns** [comment]
Nevermind, got it from the screenshot http://thirdbranch--storeido.myvtex.com/

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+- Added step css handle to login content component
```
Comment (klzns) [inline CHANGELOG.md:10]:
```suggestion
### Added
- CSS Handles for each step
```

File: react/components/LoginContent.js [hunk 2]
```diff
@@ -351,7 +352,7 @@ class LoginContent extends Component {
     )
 
     const className = classNames(
-      `${styles.content} flex relative bg-base justify-around overflow-visible pa4 center`,
+      `${styles.content} ${classNames(styles.step, `${styles.step}--${step}`)} flex relative bg-base justify-around overflow-visible pa4 center`,
```
Comment (klzns) [inline react/components/LoginContent.js:355]:
```suggestion
      `${styles.content} ${styles.step} ${styles.step}--${step} flex relative bg-base justify-around overflow-visible pa4 center`,
```

File: react/components/LoginOptions.js [hunk 3]
```diff
@@ -43,9 +43,10 @@ class LoginOptions extends Component {
       intl,
       isAlwaysShown,
       providerPasswordButtonLabel,
+      step
     } = this.props
 
-    const classes = classNames(styles.options, className, {
+    const classes = classNames(styles.options, classNames(styles.step, `${styles.step}--${step}`), className, {
```
Comment (klzns) [inline react/components/LoginOptions.js:49]:
```suggestion
    const classes = classNames(styles.options, styles.step, `${styles.step}--${step}`, className, {
```

**Comment by rafarubim** [comment]
@JNussens I replaced the `steps` handle for modifiers in the `content`, with a different name for each step. That way, if it's ever refactored, only the name mapping needs to be changed (i.e. `styleModifierByStep` obj). Does that solve the problem?

**Comment by JNussens** [comment]
@rafarubim Yes, it does. Thank you for the support!

**Comment by JNussens** [comment]
@rafarubim Do you need anything else to merge this?

**Comment by rafarubim** [comment]
I'll rebase and ask for a colleague's approval so we can release it next week!

**Comment by rafarubim** [comment]
The current changes are in
https://rafaprtest--storecomponents.myvtex.com/login

image

**Pull request merged**